src/core/gosling-component.tsx

```diff
@@ -26,7 +26,7 @@ export interface UrlToFetchOptions {
 type CompiledCallbackFn = (
     goslingSpec: gosling.GoslingSpec,
     higlassSpec: gosling.HiGlassSpec,
-    _additionalData: { _processedSpec: gosling.GoslingSpec }
+    experimental: { experimental_processedSpec: gosling.GoslingSpec }
```

Comment:
`experimental` looks duplicated to me. Since this object (processed spec) is still under `experimental`, we can just use underscore to denote that it is both experimental and internal.

```suggestion
    experimental: { _processedSpec: gosling.GoslingSpec }
```

Reply:
We can clarify this further using JS Doc that we discussed during the last meeting.

```suggestion
    /** @deprecated This is an experimental object that is not intended for production usage.
    experimental: { _processedSpec: gosling.GoslingSpec }
```